change/@azure-msal-browser-7788dd86-9af5-4992-a2d3-623b1f85ce50.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

Comment:
Do we want to mark as patch, so that browser gets a changelog entry when it consumes the updated version of common?

Reply:
Ditto with node.

Reply:
Yeah that's not a bad idea. Maybe in the future we should have our release script append "bumped @azure/msal-common to version x.x.x" and the corresponding common changelog to the bottom of the browser/node changelog so this won't be necessary for changes like this.